sce.py

```diff
@@ -1,19 +1,62 @@
 import argparse
+from subprocess import check_output, PIPE
 from tools.setup import SceSetupTool
 
+
+test_commands = {
+    'Core-v4': ['npm run api-test --prefix ./Core-v4', 'npm run lint --prefix ./Core-v4', \
+        'npm run frontend-test --prefix ./Core-v4', 'npm run build --prefix ./Core-v4'],
+    'sce-rpc': ['npm run test --prefix ./sce-rpc', 'npm run lint --prefix ./sce-rpc', \
+        'pylint * --prefix ./sce-rpc'],
+    'SCE-discord-bot': ['npm run lint --prefix ./SCE-discord-bot'],
+    'dev': ['pylint * --ignore=Core-v4,sce-rpc,SCE-discord-bot']
+}
+
 setup = SceSetupTool()
 parser = argparse.ArgumentParser()
 
-parser.add_argument('--setup', dest='setup', action='store_true')
+parser.add_argument('command', help='SCE Command')
+parser.add_argument('-p', '--project', nargs='*', help='Docker Container Name')
 args = parser.parse_args()
 
-if args.setup:
+if args.command in 'setup':
```

Comment:
revert this line

Reply:
You cant give a argument without a command. This line should stay the same

Reply:
oh i realized we changed it. lets do
```suggestion
if args.command == 'setup':
```

Reply:
Pylint also docked off points for using == which is why i changed it to 'in'

Reply:
still change it?

Reply:
lets keep it for now, but open an issue for someone to investigate why pylint is being like that

Reply:
this can be 1 issue for both problems